Blog Post on unofficial Kyma Update Twitter Bot (#746)
kyma-project/website: content/blog-posts/2021-06-28-unofficial-kyma-twitter-bot/index.md

---

> title: "Say Hello to the unofficial Kyma Update Twitter Bot"

Reviewer: ```suggestion
title: "Say hello to the unofficial Kyma Update Twitter Bot"
```

Author: Changed as proposed

author:
  name: "Christian Lechner, just another Kymma fan"
tags:
  - kmya
  - updates
  - twitter
  - community
redirectFrom:
  - "/blog/unofficial-kyma-twitter-bot"
---

> Do you spent some time on Twitter? Do you want to see a tweet when a new Kyma or Kyma CLI release was published with a reference to the release? If the answer to both is yes, then say hello to the (unofficial) Kyma Update Twitter Bot 👋.

Author: Changed as proposed

<!-- overview -->

> As you pushed the "read more" button, you would like to get informed about updates on Kyma directly via Twitter. Me too and that was the reason why I built a Twitter Bot that serves this purpose. With this short blog post I would like to give you some more information about it in an FAQ style.

Reviewer: ```suggestion
As you pushed the **Read more** button, you would like to get informed about Kyma updates directly on Twitter. Me too, and that was the reason why I built the Twitter Bot that serves this purpose. With this short blog post, I would like to give you some more information about it in FAQ style.
```

Author: Changed as proposed

> ## Which Twitter Account to follow?

Reviewer: ```suggestion
## Which Twitter account to follow?
```

Author: Changed as proposed

The account of the Twitter Bot is [@KymaUpdates](https://twitter.com/KymaUpdates).

> ## Which Releases are taken into account?

Reviewer: ```suggestion
## Which updates are taken into account?
```

Author: Changed as proposed

The bot currently only takes releases into account. Pre-releases are ignored.

> ## Which repositories are watched

Reviewer: ```suggestion
## Which repositories are watched?
```

Author: Changed as proposed

> I currently focus on Kyma and "Kyma-related" repositories. You find an up-to-date list [here](https://github.com/lechnerc77/kyma-updates-twitter-bot#watched-repositories).

Reviewer: ```suggestion
I currently focus on Kyma and Kyma-related repositories. You can find the up-to-date list [here](https://github.com/lechnerc77/kyma-updates-twitter-bot#watched-repositories).
```

Author: Changed as proposed

## How does it work?

> The main technological building blocks used behind the scenes are [Azure Functions](https://docs.microsoft.com/azure/azure-functions/) and the [Durable Extension](https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-overview). If you want to take a closer look, I have published a dedicated blog post on the building blocks. You find it [here](https://dev.to/lechnerc77/say-hello-to-the-kyma-update-twitter-bot-by-azure-durable-functions-4e1a)

Reviewer: ```suggestion
The main technological building blocks used behind the scenes are [Azure Functions](https://docs.microsoft.com/azure/azure-functions/) and the [Durable Extension](https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-overview). If you want to take a closer look, I have published a dedicated blog post on the building blocks. You can find it [here](https://dev.to/lechnerc77/say-hello-to-the-kyma-update-twitter-bot-by-azure-durable-functions-4e1a).
```

Author: Changed as proposed

> ## Where is the Code?

Reviewer: ```suggestion
## Where is the code?
```

Author: Changed as proposed

> The code is completely available on GitHub in this [repository](https://github.com/lechnerc77/kyma-updates-twitter-bot).

Reviewer: ```suggestion
The code is completely available on GitHub in [this](https://github.com/lechnerc77/kyma-updates-twitter-bot) repository.
```

Author: Changed as proposed

## Disclaimer

> The bot is still "young", so maybe it contains some bugs - so bear with my little bot. If you find a bug - pull requests are always welcome 😁.

Reviewer: ```suggestion
The bot is still "young" and it may contain some bugs, so please bear with my little bot. If you find a bug, pull requests are always welcome 😁.
```

Author: Changed as proposed
